corehq/apps/reports/standard/cases/filters.py

```diff
@@ -2,36 +2,50 @@
 from collections import Counter
 
 from django.utils.safestring import mark_safe
-from django.utils.translation import gettext_lazy
+from django.utils.translation import gettext_lazy, gettext
 from django.utils.functional import lazy
 
+from corehq.apps.accounting.utils import domain_has_privilege
 from corehq.apps.app_manager.app_schemas.case_properties import (
     all_case_properties_by_domain,
 )
 from corehq.apps.case_search.const import (
     CASE_COMPUTED_METADATA,
     SPECIAL_CASE_PROPERTIES,
+    DOCS_LINK_CASE_LIST_EXPLORER,
 )
 from corehq.apps.data_dictionary.util import get_case_property_label_dict
 from corehq.apps.data_interfaces.models import AutomaticUpdateRule
 from corehq.apps.reports.filters.base import (
     BaseSimpleFilter,
     BaseSingleOptionFilter,
 )
-from corehq import toggles
+from corehq import toggles, privileges
 
 # TODO: Replace with library method
+
+
 mark_safe_lazy = lazy(mark_safe, str)
 
 
 class CaseSearchFilter(BaseSimpleFilter):
     slug = 'search_query'
     label = gettext_lazy("Search")
-    help_inline = mark_safe_lazy(gettext_lazy(  # nosec: no user input
-        'Search any text, or use a targeted query. For more info see the '
-        '<a href="https://wiki.commcarehq.org/display/commcarepublic/'
-        'Advanced+Case+Search" target="_blank">Case Search</a> help page'
-    ))
+
+    @property
+    def help_inline(self):
+        from corehq import toggles
```

Comment:
is this line still necessary? I see that Manish added in a 'temporary' commit

Reply:
yes